.github/workflows/codeql.yml

```diff
@@ -0,0 +1,33 @@
+name: CI
+on:
+  push:
+    branches: [ master ]
+    paths-ignore:
+      - '**/*.md'
+  pull_request:
+    branches: [ master ]
+    paths-ignore:
```

Comment:
if this is for the frontend only we should probably scope it to the `www` folder.

Reply:
Done.